lib/testsuite/index.js

```diff
@@ -159,7 +159,11 @@ class TestSuite {
     }
 
     if (this.validateNightwatchInspectorCriteria()) {
```

Comment:
```javascript
const { inspectorServer, client, globalsInstance } = this;
if (this.validateNightwatchInspectorCriteria() && inspectorServer) {
  inspectorServer.closeSocket();
}

globalsInstance.inspectorServer = new NightwatchInspectorServer(client);
```

Reply:
this is logically incorrect, but I'll refactor to use optional chaining

Reply:
upon looking at it again, I don't see value in refactoring this.